lib/compiler-llvm/src/translator/code.rs

```diff
@@ -9344,6 +9344,192 @@ impl<'ctx, 'a> LLVMFunctionCodeGenerator<'ctx, 'a> {
+                    .try_as_basic_value()
+                    .left()
+                    .unwrap();
+                self.state.push1(value);
+            }
+            Operator::TableSet { table } => {
+                let table_index = self
+                    .intrinsics
+                    .i32_ty
+                    .const_int(table.into(), false)
+                    .as_basic_value_enum();
+                let (elem, value) = self.state.pop2()?;
+                let table_set = if let Some(_) = self
+                    .wasm_module
+                    .local_table_index(TableIndex::from_u32(table))
+                {
+                    self.intrinsics.table_set
+                } else {
+                    self.intrinsics.imported_table_set
+                };
+                self.builder.build_call(
+                    table_set,
+                    &[self.ctx.basic(), table_index, elem, value],
+                    "",
+                );
+            }
+            Operator::TableCopy {
+                dst_table,
+                src_table,
+            } => {
+                let (dst, src, len) = self.state.pop3()?;
+                let dst_table = self
+                    .intrinsics
+                    .i32_ty
+                    .const_int(dst_table as u64, false)
+                    .as_basic_value_enum();
+                let src_table = self
+                    .intrinsics
+                    .i32_ty
+                    .const_int(src_table as u64, false)
+                    .as_basic_value_enum();
+                self.builder.build_call(
+                    self.intrinsics.table_copy,
+                    &[self.ctx.basic(), dst_table, src_table, dst, src, len],
+                    "",
+                );
+            }
+            Operator::TableInit { segment, table } => {
+                let (dst, src, len) = self.state.pop3()?;
+                let segment = self
+                    .intrinsics
+                    .i32_ty
+                    .const_int(segment as u64, false)
+                    .as_basic_value_enum();
+                let table = self
+                    .intrinsics
+                    .i32_ty
+                    .const_int(table as u64, false)
+                    .as_basic_value_enum();
+                self.builder.build_call(
+                    self.intrinsics.table_init,
+                    &[self.ctx.basic(), table, segment, dst, src, len],
+                    "",
+                );
+            }
+            Operator::ElemDrop { segment } => {
+                let segment = self
+                    .intrinsics
+                    .i32_ty
+                    .const_int(segment as u64, false)
+                    .as_basic_value_enum();
+                self.builder.build_call(
+                    self.intrinsics.elem_drop,
+                    &[self.ctx.basic(), segment],
+                    "",
+                );
+            }
```

Comment:
Interesting, singlepass was also missing this. It must be part of the Wasm spec now though because the ref types proposal assumes it's implemented and it's not part of the ref types proposal 🤔